xml/en/docs/syntax.xml

```diff
@@ -39,18 +69,41 @@ minutes, hours, days and so on, using the following suffixes:
 </table>
 </para>
 
+</section>
+
+
+<section id="combine" name="Combining multiple units">
```

Comment:
This section discusses time intervals only.
Does it make sense to combine it with previous section?